### PR TITLE
Update: created result and question page

### PR DESCRIPTION
**Pull request by HeyNitin**
image
image

### CODE REVIEW

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me

File: Pages/results.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (ranitasahaa1230) [inline Pages/results.html:6]: You can add desc in the meta tag, like author name, page desc, page details etc

**Review by ishitakeshawani — APPROVED** [review]
LGTM

File: Pages/quiz-rules.html [hunk 2]
```diff
@@ -40,7 +40,10 @@ <h1 class="heading-sm">Rules</h1>
         I agree to above mentioned Rules.
       </label>
 
-      <a href="/pages/srk-quiz.html/" class="button btn-link"> Lets go!</a>
+
+      <a href="srk-quiz.html" class="button btn-link"> Lets go!</a>
+
+
```
Comment (smilegupta) [inline Pages/quiz-rules.html:46]: you could have formatted the code before pushing it

File: Pages/results.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    <script
+      src="https://kit.fontawesome.com/82dc64715a.js"
+      crossorigin="anonymous"
```
Comment (smilegupta) [inline Pages/results.html:9]: you can include font awesome in css file to avoid repetition for different pages

**Review by ranitasahaa1230 — APPROVED** [review]
Clean UI and also it's responsive. Looks good to me

File: Pages/srk-quiz.html [hunk 4]
```diff
@@ -4,7 +4,48 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <script
+      src="https://kit.fontawesome.com/82dc64715a.js"
+      crossorigin="anonymous"
+    ></script>
```
Comment (smilegupta) [inline Pages/srk-quiz.html:11]: Added the commnet for the same above

**Review by smilegupta — APPROVED** [review]
LGTM..Added a few sanity comments.

**Review by prakashsakari — APPROVED** [review]
Looks good to me

File: Pages/quiz-rules.html (hunk 2 above)
Comment (HeyNitin) [reply Pages/quiz-rules.html:46]: Done, Thanks for bringing it to notice.

**Pull request merged**